### DISCUSSION TITLE
Installation v3.4 on ubuntu 22.04

### DISCUSSION THREAD

**Discussion by salarihamid** [phalcon/cphalcon#16061] [General]
Hi all . is there a way to install phalcon 3.4 on the ubuntu 22 for php7.2 or php7.3 ?

the package in these php version and ppa`s are for phalcon4 not the phalcon3 .

p.s : these packages are available for the ubuntu 20 , but not the v22.x

thanks

**Comment by niden** [comment]
I do not recall whether there was a package available for that.

You can always build it directly on the machine. Check the installation section in the documents.
